tests/helpers/file-queue-helper-test.js

```diff
@@ -119,7 +119,7 @@ module('Integration | Helper | file-queue', function (hooks) {
       uploadHandler(function (/*schema, request*/) {
         // do sth
       }),
-      { timing: 2000 }
+      { timing: 200 }
```

Comment:
Two seconds was longer than the whole suite on my local. I tried reducing it and it passes locally and in CI. Seems fine 🙃